igvm_defs/src/lib.rs

```diff
@@ -628,6 +634,17 @@ pub struct IGVM_VHS_PARAMETER_AREA {
     pub file_offset: u32,
 }
 
+/// Default memory state described by the IGVM_VHT_MEMORY_STATE_PARAMETER
+/// parameter.
+#[bitfield(u32)]
+#[derive(AsBytes, FromBytes, FromZeroes, PartialEq, Eq)]
+pub struct IgvmEnvironmentInfo {
```

Comment:
can we gate this struct and other changes in igvm_defs under the unstable feature, since it's part of version 2?